helicone-node/proxy_logger/HeliconeProxyOpenAI.ts

```diff
@@ -0,0 +1,50 @@
+import { Helicone } from "../core/HeliconeOpenAIApi";
+import OpenAI from "openai";
+import * as Core from "openai/core";
+import { HeliconeHeaderBuilder } from "../core/HeliconeHeaderBuilder";
+import { IHeliconeProxyClientOptions } from "../core/HeliconeClientOptions";
+
+export class HeliconeProxyOpenAI extends OpenAI {
+  public helicone: Helicone;
+  private heliconeHeaders: { [key: string]: string };
+
+  constructor(private options: IHeliconeProxyClientOptions) {
+    const {
+      apiKey = Core.readEnv("OPENAI_API_KEY"),
+      organization = Core.readEnv("OPENAI_ORG_ID") ?? null,
```

Comment:
can we inherit this behavior from somewhere else?

Reply:
it is actually not needed as the super does that